102.md

```diff
@@ -0,0 +1,55 @@
+NIP-102
+=======
+
+Live Activities
+---------------
+
+`draft` `optional` `author:vitorpamplona`
+
+## Abstract
+
+Service providers want to offer statuses of live activities to the Nostr network in such a way that participants can be easily queried by clients. This NIP describes a general framework to advertise the involvement of pubkeys in such live activities.
+
+# Live Event
+
+A special event with kind `30311` "Live Event" is defined as a [NIP-33: Parameterized Replaceable Events](33.md) of public `p` tags. Each `p` tag SHOULD have a **displayable** marker name for the current role (e.g. `Host`, `Speaker`, `Participant`) of the user in the event and the relay information MAY be empty. This event will be constantly updated as participants join and leave the activity.
+
+For example:
+
+```json
+{
+  "kind": 30311,
+  "tags": [
+    ["d", "<unique identifier>"],
+    ["title", "<name of the event>"],
+    ["summary", "<description>"],
+    ["image", "<preview image url>"],
+    ["t", "hashtag"]
+    ["streaming", "<url>"],
+    ["starts", "<unix timestamp in seconds>"],
+    ["ends", "<unix timestamp in seconds>"],
```

Comment:
A concern I have here is for events that might have multiple occurrences, or maybe occurs across multiple days. I think I would prefer something more like:
`["time", [["<start 1 unix timestamp in seconds>", "end 1 unix timestamp in seconds"],["<start 2 unix timestamp in seconds>", "end 2 unix timestamp in seconds"]],`

Reply:
Okay, well reading NIP-01, which says: "The tags array can store a tag identifier as the first element of each subarray, plus arbitrary information afterward (always as strings)."
So, we can't have arrays as tag values.